back/db/structure.sql

```diff
@@ -1564,7 +1564,8 @@ CREATE TABLE public.phases (
     manual_votes_count integer DEFAULT 0 NOT NULL,
     manual_voters_amount integer,
     manual_voters_last_updated_by_id uuid,
-    manual_voters_last_updated_at timestamp(6) without time zone
+    manual_voters_last_updated_at timestamp(6) without time zone,
+    native_survey_method character varying
 );
```

Comment:
@jamesspeake Do I need to be concerned about the changes in this file? :thinking: Aside from the ask_follow_up, they're not related to my other changes, but doing a migrate/rebuild doesn't seem to change them..